share/spice/rfc/footer.handlebars

```diff
@@ -0,0 +1,3 @@
+<div class="one-line">
+    <div class="h-xs tx-clr--slate">RFC {{number}}</div>
```

Comment:
Great question. I had a tough time deciding where the RFC title and number should go. I thought placing the number in the footer made it more prominent. 

Here's an example in the footer:
![number-in-footer](https://cloud.githubusercontent.com/assets/129967/14723518/79fd2670-07df-11e6-88c4-40c7d971d07e.png)

And in the subtitle:
![number-in-subtitle](https://cloud.githubusercontent.com/assets/129967/14723524/7d883fb4-07df-11e6-96e5-fb082f527e62.png)